pandas/core/internals/blocks.py

```diff
@@ -3035,6 +3035,9 @@ def make_block(values, placement, klass=None, ndim=None, dtype=None,
     # For now, blocks should be backed by ndarrays when possible.
     if isinstance(values, ABCPandasArray):
         values = values.to_numpy()
+        if ndim and ndim > 1:
```

Comment:
do you need the check at all? just np.atleast_2d

Reply:
I tried that, several other tests fail if this dimensionality check is not performed.